README.md

```diff
@@ -2,18 +2,12 @@
 
```

Comment:
There is a better project template as of this morning: https://github.com/tidepool-org/data-science-project-template/blob/master/README.md

Reply:
added